amber-java-pid-follow-trajectory/src/main/java/pl/edu/agh/amber/pidfollowtrajectory/Configuration.java

```diff
@@ -0,0 +1,119 @@
+package pl.edu.agh.amber.pidfollowtrajectory;
+
+/**
+ * Created by kubicz10 on 3/11/15.
+ */
+public class Configuration {
+    private double lookahead; //maksymalna suma dlugosci rozwazanych odcinkow
+    private double iAlpha; //waga odleglosci katowej,razem z iTrack normalizowane do 1.0 przed zaaplikowaniem do wzoru na promien skretu
+    private double iTrack; //waga odleglosci liniowej, razem z iAlpha normalizowane do 1.0 przed zaaplikowaniem do wzoru na promien skretu
+    private double centreAcceleration; //maksymalne dopuszczalne przyspieszenie odsrodkowe, umozliwia pokonywanie zakretow z bezpieczna predkoscia
+    private double maxLinearVelocity; //maksymalna predkosc liniowa
+    private int bezierSegmentsNumber; //liczba odcinkow na ktore dzielony jest pojedynczy odcinek trajektoii wejsciowej przy wygladzaniu
+    private double scale; //todo: learn WTF IS THIS
```

Comment:
What is this? :smile: Anyway, you can you JavaDoc to describe that variables. It would looks like
```
public class Configuration {
    /** Maximum distance */
    private double lookahead;
}
```